src/utils/index.js

```diff
@@ -0,0 +1,24 @@
+import util from 'node-forge/lib/util';
+import HmacDrgb from 'hmac-drbg';
+import hash from 'hash.js';
+
+export const createPrng = (seed) => {
```

Comment:
I would prefer to put this in `rsa` itself as it's specific to it.

Reply:
Having that in mind, perhaps this file should be renamed to `util/binary.js`.

Reply:
Changing `createPrng` to an util simplifies testing. 
My suggestion would be to move it to `util/prng.js` and also move *typedArrayToUint8Array* to `util/binary.js`

Reply:
Alright!